opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -266,13 +297,41 @@ def _import_exporters(
     return trace_exporters, metric_exporters, log_exporters
 
 
+def _import_sampler_factory(sampler_name: str) -> Callable[[str], Sampler]:
+    _, sampler_impl = _import_config_components(
+        [sampler_name.strip()], _OTEL_SAMPLER_ENTRY_POINT_GROUP
+    )[0]
+    return sampler_impl
+
+
+def _import_sampler(sampler_name: str) -> Sampler:
```

Comment:
```suggestion
def _import_sampler(sampler_name: str) -> Optional[Sampler]:
```

Reply:
Could you explain the reasoning here? It's not like it's a network call.